src/styles/editor.scss

```diff
@@ -25,6 +25,18 @@
     border-bottom-right-radius: 8px;
   }
 
+  .border-bottom {
+    border-bottom: 1px solid $lighter-border-color;
+  }
+
+  .order1 {
+    order: 1
+  }
+
+  .order2 {
+    order: 2
```

Comment:
Codacy found an issue: [Expected a trailing semicolon (declaration-block-trailing-semicolon)](https://app.codacy.com/manual/Leecason/element-tiptap/pullRequest?prid=5779312)

Reply:
I've fixed it